### DISCUSSION TITLE
Add conditions for negative values check

### DISCUSSION THREAD

**Discussion by ShaamaHKazmi** [gorules/zen#87] [Q&A]
Hi, I'm completely new to Zen based engine. I'm trying to check in the my decision table when an input (negative value) is lower than the negative value in the column by using the expression <-2 for e.g. But the comparison doesn't seem to work for negative values. Maybe this way of writing expression is not suited for negative values. Any idea how I can make it work? Thanks in advance!

**Comment by stefan-gorules** [comment]
Confirmed issue, looking into it 👀

**Comment by stefan-gorules** [comment]
@ShaamaHKazmi Thanks for reporting, after we've rewritten major parts of zen-expression for better maintainability, the issue is now solved.
